src/Microsoft.AspNetCore.SignalR.Redis/Internal/RedisLoggerExtensions.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.SignalR.Redis.Internal
+{
+    internal static class RedisLoggerExtensions
+    {
+        // Category: RedisHubLifetimeManager<THub>
+        private static readonly Action<ILogger, string, Exception> _connectingToEndpoints =
+            LoggerMessage.Define<string>(LogLevel.Information, new EventId(0, nameof(ConnectingToEndpoints)), "Connecting to redis endpoints: {endpoints}.");
+
+        private static readonly Action<ILogger, Exception> _connected =
+            LoggerMessage.Define(LogLevel.Information, new EventId(1, nameof(Connected)), "Connected to redis.");
+
+        private static readonly Action<ILogger, string, Exception> _subscribing =
+            LoggerMessage.Define<string>(LogLevel.Information, new EventId(2, nameof(Subscribing)), "Subscribing to channel: {channel}.");
+
+        private static readonly Action<ILogger, string, Exception> _receivedFromChannel =
+            LoggerMessage.Define<string>(LogLevel.Trace, new EventId(3, nameof(ReceivedFromChannel)), "Received message from redis channel {channel}.");
+
+        private static readonly Action<ILogger, string, Exception> _publishToChannel =
+            LoggerMessage.Define<string>(LogLevel.Trace, new EventId(4, nameof(PublishToChannel)), "Publishing message to redis channel {channel}.");
+
+        private static readonly Action<ILogger, string, Exception> _unsubscribe =
+            LoggerMessage.Define<string>(LogLevel.Information, new EventId(5, nameof(Unsubscribe)), "Unsubscribing from channel: {channel}.");
```

Comment:
Not sure about this subscribe and unsubscribe being information

Reply:
Debug or Trace?

Reply:
I would go with Trace